### ISSUE TITLE
Mystic's vow of rebirth targets non-party members

### ISSUE THREAD

**Issue by Owyn** [baldera-mods/lockons#1]
Mystic's vow of rebirth targets non-party members

**Comment by meishuu** [comment]
Yeah, I was going to get around to fixing that, Immersion, and Divine Intervention at some point. Thank you for opening the issue, though.

**Comment by Uwuu** [comment]
Any update yet? Immersion and divine still targeting people not in the same party and it can get annoying sometimes :/

**Comment by Owyn** [comment]
I've forked it and made a lighter version of lockons (it caches server responses instead of saying that you can lockon anything instantly), it has no such issue (unless you locked onto those players before)
https://github.com/Owyn/lockons-light
